cmd/containerd/builtins_cri.go

```diff
@@ -1,3 +1,4 @@
+// +build linux windows
```

Comment:
I checked https://github.com/kunalkushwaha/ltag/blob/master/golang.go#L19 and found that `ltag` only checks one line build option.

The workaround  is that `+build linux,!no_cri windows,!no_cri`

Reply:
cc @kunalkushwaha